src/api/DesktopAgent.ts

```diff
@@ -13,7 +13,6 @@ import { ImplementationMetadata } from './ImplementationMetadata';
 import { PrivateChannel } from './PrivateChannel';
 import { AppIdentifier } from './AppIdentifier';
 import { AppMetadata } from './AppMetadata';
-import { DesktopAgentDetails } from './GetAgent';
```

Comment:
Not actually used or re-exported - so yes this should go.

However I think we're missing a separate export of everything from GetAgent in index.ts... Probably need to add:

```ts
export * from './api/GetAgent';
```

Reply:
I've added that